packages/cli/src/services/communityPackages.service.ts

```diff
@@ -221,7 +223,7 @@ export class CommunityPackageService {
 	}
 
 	hasPackageLoaded(packageName: string) {
-		const missingPackages = config.get('nodes.packagesMissing') as string | undefined;
+		const missingPackages = config.getEnv('nodes.packagesMissing') as string | undefined;
```

Comment:
```suggestion
		const missingPackages = config.getEnv('nodes.packagesMissing');
```

Unless this is intentional and `undefined` can somehow end up here?